doc/man/restic-diff.1

```diff
@@ -10,7 +10,7 @@ restic\-diff \- Show differences between two snapshots
 
 .SH SYNOPSIS
 .PP
-\fBrestic diff snapshot\-ID snapshot\-ID [flags]\fP
+\fBrestic diff [flags] snapshot\-ID snapshot\-ID\fP
```

Comment:
Please remove all man-page changes, these are generated automatically on release.

Reply:
Okay, that makes sense.